build.sbt

```diff
@@ -21,7 +21,7 @@ libraryDependencies ++= Seq(
   javaWs,
   javaJpa,
   "org.eclipse.persistence" % "eclipselink" % "2.6.2",
-  "com.github.ONSdigital" % "dp-dd-backend-model" % "develop-SNAPSHOT",
+  "com.github.ONSdigital" % "dp-dd-backend-model" % "feature~add_dimension_table-SNAPSHOT",
```

Comment:
This should change back to use develop-SNAPSHOT once the model changes have been merged.